src/main/java/com/google/api/codegen/transformer/StaticLangApiMethodTransformer.java

```diff
@@ -41,18 +36,18 @@
 import com.google.api.codegen.viewmodel.UnpagedListCallableMethodDetailView;
 import com.google.api.tools.framework.model.Field;
 import com.google.api.tools.framework.model.TypeRef;
-import com.google.common.base.Predicate;
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableMap;
-import com.google.common.collect.Iterables;
-import com.google.protobuf.DescriptorProtos.FieldDescriptorProto.Type;
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
 
-/** ApiMethodTransformer generates view objects from method definitions. */
-public class ApiMethodTransformer {
+/**
+ * StaticLangApiMethodTransformer generates view objects from method definitions in static
+ * languages.
+ */
+public class StaticLangApiMethodTransformer {
```

Comment:
Should private method 'getMethodParamDocs' in this class be part of separate ParamDocTransformer implementation such as StaticLangParamDocTransformer? This makes the use of ParamDocTransformer consistent across all languages.